tests/event_seg/test_event.py

```diff
@@ -0,0 +1,29 @@
+import brainiak.event_seg.event
+import numpy as np
+
+def test_create_event_segmentation():
+    es = brainiak.event_seg.event.EventSegment(5)
+    assert es, "Invalid EventSegment instance"
+
+
+def test_events():
+    K = 5
+    V = 3
+    T = 10
+    es = brainiak.event_seg.event.EventSegment(K, max_steps=2)
+    sample_data = np.random.rand(V, T)
+    segments = es.learn_events(sample_data)
+
+    assert segments.shape == (T,K), "Segmentation from learn_events has " \
+                                    "incorrect shape"
+    assert np.isclose(np.sum(segments, axis=1), np.ones(T)).all(), \
+        "Segmentation from learn_events not correctly normalized"
+
+    T2 = 15
+    sample_data2 = np.random.rand(V, T2)
+    test_segments, test_ll = es.find_events(sample_data2)
+
+    assert test_segments.shape == (T2, K), "Segmentation from find_events " \
+                                           "has incorrect shape"
+    assert np.isclose(np.sum(test_segments, axis=1), np.ones(T2)).all(), \
+        "Segmentation from find_events not correctly normalized"
```

Comment:
You should add a simple manually-constructed example with well-understood results and test that you get those results.